pkg/controllers/disruption/controller.go

```diff
@@ -184,7 +185,8 @@ func (c *Controller) executeCommand(ctx context.Context, m Method, cmd Command)
 		methodLabel:            m.Type(),
 		consolidationTypeLabel: m.ConsolidationType(),
 	}).Inc()
-	logging.FromContext(ctx).Infof("disrupting via %s %s", m.Type(), cmd)
+	commandID := uuid.NewUUID()
+	logging.FromContext(ctx).With("command-id", commandID).Infof("disrupting via %s %s", m.Type(), cmd)
 
 	stateNodes := lo.Map(cmd.candidates, func(c *Candidate, _ int) *state.StateNode {
 		return c.StateNode
```

Comment:
Do any of the errors that are fired in this block also need to have the `command-id` here?

Reply:
I added it in the error log lines. Lmk what you think